tools/pipelines/templates/build-npm-package.yml

```diff
@@ -280,6 +285,23 @@ stages:
               tagName: ${{ parameters.tagName }}
               interdependencyRange: ${{ parameters.interdependencyRange }}
 
+        # Generate rollups and update package.json types to alpha/beta
+        - ${{ if or(eq(variables.releaseAlphaBetaBuild, 'alpha'), eq(variables.releaseAlphaBetaBuild, 'beta')) }}:
+          - task: Npm@1
+            displayName: npm run ${{ parameters.taskGenerateRollups }}
+            inputs: 
+              command: 'custom'
+              workingDir: ${{ parameters.buildDirectory }}
+              customCommand: 'run ${{ parameters.taskGenerateRollups }}'
```

Comment:
As noted earlier this shouldn't be needed, because the build will do the docs build anyway.